packages/rocketchat-slider/rocketchat-slider.scss

```diff
@@ -0,0 +1,112 @@
+$shade-10: #04436a !default;
```

Comment:
We don't use scss, just cssnext and less(we will remove all less styles in the future). You need to add your style on rocketchat-theme package and set your custom colors there on variables.js and colors.less files.